homeassistant/components/google_generative_ai_conversation/entity.py

```diff
@@ -312,7 +312,9 @@ def __init__(self, entry: ConfigEntry, subentry: ConfigSubentry) -> None:
             identifiers={(DOMAIN, subentry.subentry_id)},
             name=subentry.title,
             manufacturer="Google",
-            model="Generative AI",
+            model=subentry.data.get(CONF_CHAT_MODEL, RECOMMENDED_CHAT_MODEL).split("/")[
```

Comment:
This won't work for the TTS entity as it has a different recommended model.

Reply:
Ah, last time I checked TTS wasn't inheriting this class

Reply:
Maybe add `default_model:str = RECOMMENDED_CHAT_MODEL` in the init and pass `RECOMMENDED_TTS_MODEL` from the TTS init?
While you are at it, change the other usage of RECOMMENDED_CHAT_MODEL in _async_handle_chat_log in this class to use `self.default_model`. The TTS entity doesn't use that function but it would be good to change it just in case a future subclass does.